lib/std/zig.zig

```diff
@@ -9,6 +11,21 @@ pub const ast = @import("zig/ast.zig");
 pub const system = @import("zig/system.zig");
 pub const CrossTarget = @import("zig/cross_target.zig").CrossTarget;
 
+pub const SrcHash = [16]u8;
+
+/// If the source is small enough, it is used directly as the hash.
```

Comment:
this seems like a recipe for collisions. blake3 should be fast enough that we should play it safe...

Reply:
Ahh, because a real hash will collide with the source?